tests/features/activity/activity.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { expect } from '@playwright/test';
+import { test } from '../../config/test-base';
+
+const entityNameWithAlphabeticChars = 'Book_ETL_aqa'
+
+test.describe('Check filters', () => {
+    test.describe('When apply single filter', () => {
+        test.beforeEach(async ({steps: {pages}, page}) => {
+            await test.step(`I open activity page`, async () => {
+                await page.goto('');
+                await pages.topPanel.clickTab('Activity');
+            });
+        });
+        /**
+         * /project/1/test-cases/139
+         * /project/1/test-cases/141
+         * /project/1/test-cases/142
+         */
+        const parameters = [
+            { testName: 'Event type', filterName: 'Event type', inputValue: 'Data Entity Created' },
+            { testName: 'Datasource', filterName: 'Datasource', inputValue: 'Book_ETL_aqa' },
+            { testName: 'Namespace', filterName: 'Namespace', inputValue: 'ETL' },
+        ];
+        for (const parameter of parameters) {
+            test(`Display an expected item with filter ${parameter.testName}`, async ({
+                                                                                               steps: {pages},
+                                                                                           }) => {
+                await test.step(`When apply filter ${parameter.testName}`, async () => {
+                    await pages.activity.openFilterWithSelect(`${parameter.filterName}`);
+                    await pages.activity.chooseOption(`${parameter.inputValue}`)
+                    expect(await pages.activity.countListItems(entityNameWithAlphabeticChars)).toBeGreaterThanOrEqual(1);
+                });
+            });
+        };
+        /**
+         * /project/1/test-cases/143
+         */
+            test(`Display an expected item with filter Tag`, async ({
+                                                                                          steps: {pages},
+                                                                                      }) => {
+                await test.step(`When apply filter Tag`, async () => {
+                    await pages.activity.searchByTextInFilter(`Tag`, 'aqa');
+                    expect(await pages.activity.countListItems(entityNameWithAlphabeticChars)).toBeGreaterThanOrEqual(1);
+                });
+            });
+        /**
+         * /project/1/test-cases/144
+         */
+        test(`Display an expected item with filter Owner`, async ({workerId,
+                                                                    steps: {pages},
+                                                                }) => {
+            const ownerName7 = `ownerName7 + ${workerId}`;
```

Comment:
pls use 
```
const ownerName7 = `ownerName7${workerId}`;
```
instead of 
```
const ownerName7 = `ownerName7 + ${workerId}`;
```

I see this pattern in multiple places in your PR